### DISCUSSION TITLE
Documentation : File Upload + Static in the same component

### DISCUSSION THREAD

**Discussion by MGeurts** [tasteui/tasteui#390] [Ideas]
It would be great to have a working demo on the [https://tallstackui.com/docs/form/upload](https://tallstackui.com/docs/form/upload) topic having in the same component:

1. the static method showing existing images with the option to delete individual images
2. the possibility to upload (add) one or multiple images

Thanks.

**Comment by devajmeireles** [comment]
Hey, @MGeurts ! The upload component in static mode does not support uploading files at the same time.

I suggest you use an if condition to show the upload component in static mode to receive new files after deleting all other files using the component in static mode.

```blade
@if (count($staticFiles) > 0)
  {{-- show the upload component in static mode --}}
@else
  {{-- show the normal upload component --}}
```
